.travis.yml

```diff
@@ -66,6 +80,11 @@ deploy:
     skip_cleanup: true
     on:
       tags: true
+  - provider: script
+    script: ./gradlew managementportal-client:bintrayUpload
```

Comment:
should this be `:managementportal-client:bintrayUpload`? (note the first colon `:`)